### PR TITLE
Completed css-brochure-form exercise.

### PR DESCRIPTION
**Pull request by ChorongShin**
Screen Shot 2022-01-26 at 9 05 51 PM

### CODE REVIEW

**Review by codingcodymiller — CHANGES_REQUESTED** [review]
Hey Stella! Your submit button is too close to the select elem ent above it as compared to the example, please revise and submit a new screenshot with your revision.

**Comment by ChorongShin** [comment]
Screen Shot 2022-01-27 at 11 43 34 AM

**Review by codingcodymiller — CHANGES_REQUESTED** [review]
Hey Stella! Now, your submit button is jutting out past the end of the select on the right side, please revise and submit a new screenshot with your revision.

**Comment by ChorongShin** [comment]
Screen Shot 2022-01-28 at 11 14 32 AM

File: css-brochure-form/styles.css [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+}
+
+.button button {
+  display: inline-block;
+  text-align: center;
+}
+
+div.button {
+  text-align: right;
+  margin-right: 18%;
```
Comment (codingcodymiller) [inline css-brochure-form/styles.css:67]: Oh no!!! That's a **_lot_** of margin. Why is this necessary? There is likely a better way to accomplish this.

File: css-brochure-form/styles.css [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+  padding-right: 17px;
+}
+
+div.center {
+  padding-top: 0.5px;
+  text-align: center;
+}
+
+div.left {
+  margin-left: 153px;
```
Comment (codingcodymiller) [inline css-brochure-form/styles.css:78]: Oh no!!! That's a **_lot_** of margin. Why is this necessary? There is likely a better way to accomplish this.

**Review by codingcodymiller — CHANGES_REQUESTED** [review]

**Comment by ChorongShin** [comment]
Screen Shot 2022-01-28 at 3 49 18 PM

File: css-brochure-form/styles.css [hunk 3]
```diff
@@ -64,9 +64,8 @@ button {
 
 div.button {
   text-align: right;
-  margin-right: 18%;
   padding-top: 10px;
-  padding-right: 17px;
+  padding-right: 138px;
```
Comment (codingcodymiller) [inline css-brochure-form/styles.css:68]: You can't just change the `margin` to `padding`, that is the same problem.

File: css-brochure-form/styles.css [hunk 4]
```diff
@@ -75,7 +74,7 @@ div.center {
 }
 
 div.left {
-  margin-left: 153px;
+  padding-left: 153px;
```
Comment (codingcodymiller) [inline css-brochure-form/styles.css:77]: You can't just change the `margin` to `padding`, that is the same problem.

**Review by codingcodymiller — CHANGES_REQUESTED** [review]

**Comment by ChorongShin** [comment]
Screen Shot 2022-02-01 at 9 54 40 AM

File: css-brochure-form/styles.css [hunk 5]
```diff
@@ -4,15 +4,35 @@ form {
...
 
-h3 {
+h2 {
   font-family: "Brandon Grotesque", sans-serif;
   color: #fc6b62;
-  text-align: center;
   font-weight: bold;
+  text-align: center;
+  position: relative;
+  right: 123px;
```
Comment (codingcodymiller) [inline css-brochure-form/styles.css:35]: Why is this necessary? You already have `text-align: center` on this 🤔

File: css-brochure-form/styles.css [hunk 6]
```diff
@@ -55,26 +76,14 @@ button {
...
-
-div.center {
-  padding-top: 0.5px;
-  text-align: center;
-}
-
-div.left {
-  padding-left: 153px;
+  position: relative;
+  left: 100px;
```
Comment (codingcodymiller) [inline css-brochure-form/styles.css:86]: You seem to be becoming **_very_** reliant on the `position` property. Let me reiterate that it is meant to be used **_very sparingly_**, and most often for very small adjustments, something like 5-10px. What is the purpose of this?

**Review by codingcodymiller — CHANGES_REQUESTED** [review]

**Comment by ChorongShin** [comment]
Screen Shot 2022-02-01 at 6 25 59 PM

**Review by codingcodymiller — APPROVED** [review]
Great work!!

**Pull request merged**